Kernel/Tasks/Process.h

```diff
@@ -129,6 +129,7 @@ class Process final
         Atomic<u32> thread_count { 0 };
         u8 termination_status { 0 };
         u8 termination_signal { 0 };
+        UnixDateTime creation_time;
```

Comment:
What's the rationale for making this a protected process value?

Reply:
I put it there purely because I expect the value not to change after it's initially set. From a security point of view, I don't think there's anything catastrophic that can be done by changing the value, although I'm sure a bad actor could think of something.

Do you think it would be better to make this a normal member variable instead?

Reply:
I don't see a reason to put this as a protected process value. UID and GID values are needed to be there because changing them could lead to privilege escalation. I can't see how changing the `creation_time` would lead to a security problem.

Reply:
I've now changed `creation_time` to be a normal member variable instead of a protected value.